source/getting-started/index.markdown

```diff
@@ -57,7 +57,7 @@ We will need a few things to get started with installing Home Assistant. The lat
    - Create a folder named `network` in the root of the newly formatted USB-stick.
    - Within that folder create a file named `my-network` without extension.
    - Copy one of [the examples] to the `my-network` file and adjust accordingly.
-   - Plug the USB-stick into the Raspberry Pi 3.
+   - Plug the USB-stick into the Raspberry Pi 3. (The USB-stick needs to be inserted to the device during the following setup process only and can be disconnected afterwards.)
```

Comment:
I kinda think this is obvious, way to much text at this part and in a way: not true.

The network configuration is read early in the process (not that it really matters, but that is just details).

Reply:
Maybe I'm not the average user here but when I set up Hass.IO I was desperately looking for some USB drive I would not need later on. I only read later that it's not persistent.
Also, here:
https://community.home-assistant.io/t/hassos-wlan/59988?
https://community.home-assistant.io/t/hass-io-wifi/59778/2?

Reply:
Well maybe add an additional step at the end:

```Markdown
1. If you have used a USB-stick for configuring the network, it can now be removed.
```

Which is much cleaner and follow a step-by-step procedure, instead of trying to squeeze in a note in a step in the middle.